index.html

```diff
@@ -32,7 +32,7 @@
   <div id="main">
     <span style="font-size:100px;cursor:pointer" onclick="toggleNav()">&#9881;</span>
     <span style="font-size:100px;cursor:pointer" onclick="openCreateContactModal()">&plus;</span>
-    <span id="userName"></span><br/>
+    <!-- <span id="userName"></span><br/> -->
```

Comment:
Please add a TODO comment above the commented out line so we know to come back to it later